python/adios2/io.py

```diff
@@ -103,10 +103,7 @@ def available_attributes(self):
                 value
                     attribute information dictionary
         """
-        attributes = {}
-        for name, attr in self.impl.AvailableAttributes():
-            attributes[name] = Attribute(attr, name)
-        return attributes
+        return self.impl.AvailableAttributes()
```

Comment:
This will return binding attributes rather than High Level API attributes. Note that we want the user not to use the bindings unless he wants by using the impl member variable. Also note that other hight level python classes expect the high level attributes rather than the bindings for it to work

Reply:
Actually, this function returns a dictionary of dictionaries with the attribute names as key and information about each attribute as value. It's the inquire_variable() function that returns a high-level attribute, which I did not change. 

The existing loop simply fails as the dictionary cannot be converted to an attribute but is also shouldn't be.